src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$fontSize: 30px;
+$fontFamily: arial, helvetica, sans-serif;
+$fontColor: #000;
+
+$cell-size: 100px;
+$calendar-gap: 1px;
+$calendar-width: $cell-size * 7 + 6 * $calendar-gap;
+$start-day:
+  "mon" 0,
+  "tue" calc(#{$cell-size} + #{$calendar-gap}),
+  "wed" calc(2 * (#{$cell-size} + #{$calendar-gap})),
```

You can calculate this in one time instead calculate each time
Also, you don't need `calc` for calculation in scss
```suggestion
$calendar-width: $cell-size * 7 + 6 * $calendar-gap;
$day-margin: $cell-size + $calendar-gap;
$start-day:
  "mon" 0,
  "tue" $day-margin,
  "wed" 2 * $day-margin,
```